src/wrappers/themis/react-native-themis/ios/RCTThemis.m

```diff
@@ -89,10 +89,13 @@ + (NSData*)dataDeserialize:(NSArray*) data
         @throw exception;
     }
 
+    NSNumber *uchar_min = [NSNumber numberWithInt:0];
+    NSNumber *uchar_max = [NSNumber numberWithInt:255];
```

Comment:
These could be static constants, avoiding initialization on every call.

Reply:
Obj-c do not compile line like : const NSNumber *myvar = [NSNumber numberWithInt:0]
So I initialized these properties only once in init. 
Do you accept it?

Reply:
> Do you accept it?

Yes.

<sub>I thought about `static` but I forgot whether Objective-C guarantees that initialization to be thread-safe, etc. Since we have that giant singleton anyway, might as well using it for globals.</sub>